src/main/scala/advancepassengerinfo/importer/DqApiFeed.scala

```diff
@@ -42,8 +45,15 @@ case class DqApiFeedImpl(fileNamesProvider: FileNames,
               metricsCollector.counter("api-dq-zip-failure", 1)
           }
           .map(_ => zipFileName)
-      }
-      .wireTap(_ => metricsCollector.counter("api-dq-zip-processed", 1))
+      }.recoverWithRetries(3, { case t =>
```

Comment:
Again, we could do with a test that triggers this new `recoverWithRetries`. It might be that we have covered all failures already in which case this code isn't reachable, but we need to check

Reply:
Add test for this one . I can see no logs are present we don't have recover and log .